squareone.yml

```diff
@@ -0,0 +1,6 @@
+# SquareOne Project Docker Config
```

Comment:
Let's add a bit more clarity here about how this can be customized and why.

e.g. something like: run `so config:copy` and move the `/home/$USER/.config/squareone/squareone.yml` to override this file to fully customize docker for your project.